### PR TITLE
Add GitHub Actions examples

### PR DESCRIPTION
**Pull request by chris-codeflow**
- Added GitHub Actions examples (aligned with the Azure Pipelines examples) for the setup and execute tasks.  Verified the examples using a test workflow.

### Related changes
- Added GitHub release badge for the latest release to the README file.
- Added missing **gitversion/execute** action outputs to the action definition and model.

### Other changes
- Changed the logo for the Azure Pipelines marketplace badge to the Azure Pipelines logo (from Visual Studio).

### CODE REVIEW

**Comment by chris-codeflow** [comment]
@arturcic .  PR ready for review.

File: docs/examples/azure/gitversion/execute/usage-examples.md [hunk 1]
```diff
@@ -66,8 +66,8 @@ The Execute GitVersion task creates the following job-scoped variables and multi
 - GitVersion.NuGetPreReleaseTag
 - GitVersion.VersionSourceSha
 - GitVersion.CommitsSinceVersionSource
-- GitVersion.CommitsSinceVersionSourcePadded
-- GitVersion.UncommittedChanges
+- GitVersion.CommitsSinceVersionSourcePadded (since 5.2.0)
+- GitVersion.UncommittedChanges (since 5.5.0)
```
Comment (arturcic) [inline docs/examples/azure/gitversion/execute/usage-examples.md:70]: I like this, the version when the variable was added

**Review by arturcic — APPROVED** [review]

**Comment by arturcic** [comment]
@chris-codeflow that is amazing, thanks for adding the samples for github actions as well as for azure pipelines 👍

**Comment by chris-codeflow** [comment]
No problem at all @arturcic .  I've got some improvements to make (such as using a heading for each example, to make them easier to reference) which I will submit under another PR.

**Comment by arturcic** [comment]
Great, then we can release 0.9.7 with the updated docs. Maybe it makes sense to update the docs with version 0.9.7 in the other PR so that we will not need a separate commit after the release.

**Comment by chris-codeflow** [comment]
Hello @arturcic .  The commit to fix the spelling of omitted has accidentally broken some references to the 'Uncommitted...' output.  I will fix this and submit another PR.

Sorry.

**Comment by chris-codeflow** [comment]
@arturcic .  One more thing, I would like to change the **use-actions** GitHub Action name to **gittools-actions** (same as the Azure Pipelines task) as it's not intuitive at the moment.

What are your thoughts on this?

**Comment by arturcic** [comment]
I wonder if this will not break existing workflows that use the current name

**Comment by chris-codeflow** [comment]
I think it would be ok, as the workflow YAML files reference the entry points (no reference to use-actions).I'm presuming it would orphan the existing 'use-actions' action on the GitHub marketplace though.We could at least remove the inputs, which are for the gitversion/setup action specifically, not for the other actions.   When I created a new workflow and added the use-actions action, it added the inputs which I promptly removed!Regards,Chris ---- On Sun, 22 Nov 2020 10:54:02 +0000  notifications@github.com  wrote ----
I wonder if this will not break existing workflows that use the current name

**Comment by arturcic** [comment]
Ok let's rename it. We'll need to update the readme file here as well as on the gitversion repo to point to the correct GitHub actions

**Comment by chris-codeflow** [comment]
Thanks @arturcic .  I will submit one PR for the improved examples and another for the action name change (inc. README updates).  I will try and submit this evening.

**Pull request merged**